src/lib.rs

```diff
@@ -203,6 +246,7 @@ impl<T: Runtime> Clone for Client<T> {
             runtime_version: self.runtime_version.clone(),
             _marker: PhantomData,
             page_size: self.page_size,
+            signed_options: self.signed_options.clone(),
```

Comment:
The type implements `Copy`

```suggestion
            signed_options: self.signed_options,
```